lib/open_food_network/order_cycle_management_report.rb

```diff
@@ -83,7 +83,7 @@ def filter_to_payment_method(orders)
 
     def filter_to_shipping_method(orders)
       if params[:shipping_method_in].present?
-        orders.joins(:shipping_method).where("spree_shipping_methods.name = ?", params[:shipping_method_in])
+        orders.joins(:shipping_method).where(shipping_method_id: params[:shipping_method_in])
```

Comment:
allows user to select multiple shipping methods to filter by (and filters by id rather than name)